app/factories/LobbyFactory.js

```diff
@@ -8,11 +9,24 @@ function LobbyFactory(){
     var lobby;
 
     lobby = {
-      name: options.name || "",
-      hostSettings: options.hostSettings || [],
-      playerSettings: options.playerSettings || []
+      //Just for test we disable this
+      //name: options.name || "",
+      //hostSettings: options.hostSettings || [],
```

Comment:
You should either remove these or leave them uncommented. I'm guessing you commented these lines because you didn't know how they work, which isn't really a valid solution.